examples/ipython/LaTeX.ipynb

```diff
@@ -1535,21 +1535,20 @@
\\left ( e_{v}\\cdot e_{v}\\right ) ^{2} - 2 \\left ( e_{u}\\cdot e_{u}\\right )  \\left ( e_{u}\\cdot e_{v}\\right ) ^{2} \\left ( e_{v}\\cdot e_{v}\\right )  + \\left ( e_{u}\\cdot e_{v}\\right ) ^{4}} \\boldsymbol{e}_{u} + \\frac{- \\left ( e_{u}\\cdot e_{u}\\right )  \\left ( e_{u}\\cdot e_{v}\\right )  A_{vv} B_{vv} + \\left ( e_{u}\\cdot e_{u}\\right )  \\left ( e_{v}\\cdot e_{v}\\right )  A_{vv} B_{vu} + \\left ( e_{u}\\cdot e_{v}\\right ) ^{2} A_{vu} B_{vv} + \\left ( e_{u}\\cdot e_{v}\\right ) ^{2} A_{vv} B_{uv} - \\left ( e_{u}\\cdot e_{v}\\right )  \\left ( e_{v}\\cdot e_{v}\\right )  A_{vu} B_{uv} - \\left ( e_{u}\\cdot e_{v}\\right )  \\left ( e_{v}\\cdot e_{v}\\right )  A_{vu} B_{vu} - \\left ( e_{u}\\cdot e_{v}\\right )  \\left ( e_{v}\\cdot e_{v}\\right )  A_{vv} B_{uu} + \\left ( e_{v}\\cdot e_{v}\\right ) ^{2} A_{vu} B_{uu}}{\\left ( e_{u}\\cdot e_{u}\\right ) ^{2} \\left ( e_{v}\\cdot e_{v}\\right ) ^{2} - 2 \\left ( e_{u}\\cdot e_{u}\\right )  \\left ( e_{u}\\cdot e_{v}\\right ) ^{2} \\left ( e_{v}\\cdot e_{v}\\right )  + \\left ( e_{u}\\cdot e_{v}\\right ) ^{4}} \\boldsymbol{e}_{v} \\\\ L \\left ( \\boldsymbol{e}_{v}\\right ) =& \\frac{\\left ( e_{u}\\cdot e_{u}\\right ) ^{2} A_{uv} B_{vv} - \\left ( e_{u}\\cdot e_{u}\\right )  \\left ( e_{u}\\cdot e_{v}\\right )  A_{uu} B_{vv} - \\left ( e_{u}\\cdot e_{u}\\right )  \\left ( e_{u}\\cdot e_{v}\\right )  A_{uv} B_{uv} - \\left ( e_{u}\\cdot e_{u}\\right )  \\left ( e_{u}\\cdot e_{v}\\right )  A_{uv} B_{vu} + \\left ( e_{u}\\cdot e_{u}\\right )  \\left ( e_{v}\\cdot e_{v}\\right )  A_{uu} B_{uv} + \\left ( e_{u}\\cdot e_{v}\\right ) ^{2} A_{uu} B_{vu} + \\left ( e_{u}\\cdot e_{v}\\right ) ^{2} A_{uv} B_{uu} - \\left ( e_{u}\\cdot e_{v}\\right )  \\left ( e_{v}\\cdot e_{v}\\right )  A_{uu} B_{uu}}{\\left ( e_{u}\\cdot e_{u}\\right ) ^{2} \\left ( e_{v}\\cdot e_{v}\\right ) ^{2} - 2 \\left ( e_{u}\\cdot e_{u}\\right )  \\left ( e_{u}\\cdot e_{v}\\right ) ^{2} \\left ( e_{v}\\cdot e_{v}\\right )  + \\left ( e_{u}\\cdot e_{v}\\right ) ^{4}} \\boldsymbol{e}_{u} + \\frac{\\left ( e_{u}\\cdot e_{u}\\right ) ^{2} A_{vv} B_{vv} - \\left ( e_{u}\\cdot e_{u}\\right )  \\left ( e_{u}\\cdot e_{v}\\right )  A_{vu} B_{vv} - \\left ( e_{u}\\cdot e_{u}\\right )  \\left ( e_{u}\\cdot e_{v}\\right )  A_{vv} B_{uv} - \\left ( e_{u}\\cdot e_{u}\\right )  \\left ( e_{u}\\cdot e_{v}\\right )  A_{vv} B_{vu} + \\left ( e_{u}\\cdot e_{u}\\right )  \\left ( e_{v}\\cdot e_{v}\\right )  A_{vu} B_{uv} + \\left ( e_{u}\\cdot e_{v}\\right ) ^{2} A_{vu} B_{vu} + \\left ( e_{u}\\cdot e_{v}\\right ) ^{2} A_{vv} B_{uu} - \\left ( e_{u}\\cdot e_{v}\\right )  \\left ( e_{v}\\cdot e_{v}\\right )  A_{vu} B_{uu}}{\\left ( e_{u}\\cdot e_{u}\\right ) ^{2} \\left ( e_{v}\\cdot e_{v}\\right ) ^{2} - 2 \\left ( e_{u}\\cdot e_{u}\\right )  \\left ( e_{u}\\cdot e_{v}\\right ) ^{2} \\left ( e_{v}\\cdot e_{v}\\right )  + \\left ( e_{u}\\cdot e_{v}\\right ) ^{4}} \\boldsymbol{e}_{v}  \\end{array} \\right \\} \\end{equation*}\n",
```

Comment:
This one was removed - sympy 1.3 decided to do some substantial `-` rearrangment compared to 1.4 and 1.5. Fixing that with a regex is totally non-viable for a 64kB line.